pkg/ddevapp/config.go

```diff
@@ -1412,8 +1412,8 @@ func DiscoverDefaultDocroot(app *DdevApp) string {
 func (app *DdevApp) docrootPrompt() error {
 
 	// Determine the document root.
-	util.Warning("\nThe docroot is the directory from which your site is served.\nThis is a relative path from your project root at %s", app.AppRoot)
-	output.UserOut.Println("You may leave this value blank if your site files are in the project root")
+	output.UserOut.Printf("\nThe docroot is the directory from which your site is served.\nThis is a relative path from your project root at %s\n", app.AppRoot)
+	output.UserOut.Printf("Leave docroot empty (hit <RETURN>) if your index.php is in the project root.\n")
```

Comment:
Maybe this is better?:

```suggestion
	output.UserOut.Printf("Leave docroot empty (hit <RETURN>) to use the location shown in parentheses.\nOr specify a custom path if your index.php is in a different directory.\n")
```

We need to emphasize that the default value is in parentheses.